lib/isaac.go

```diff
@@ -67,23 +67,23 @@ func (rv *RoundVote) GetResult(state sebakcommon.BallotState) (result RoundVoteR
 	return result
 }
 
-func (rv *RoundVote) CanGetVotingResult(policy sebakcommon.VotingThresholdPolicy, state sebakcommon.BallotState) (RoundVoteResult, VotingHole, bool) {
+func (rv *RoundVote) CanGetVotingResult(policy sebakcommon.VotingThresholdPolicy, state sebakcommon.BallotState) (RoundVoteResult, sebakcommon.VotingHole, bool) {
 	threshold := policy.Threshold(state)
 	if threshold < 1 {
-		return RoundVoteResult{}, VotingNOTYET, false
+		return RoundVoteResult{}, sebakcommon.VotingNOTYET, false
 	}
 
 	result := rv.GetResult(state)
 	if len(result) < int(threshold) {
-		return result, VotingNOTYET, false
+		return result, sebakcommon.VotingNOTYET, false
 	}
 
-	var yes int
-	var no int
-	for _, vh := range result {
-		if vh == VotingYES {
+	var yes, no int
+	for _, votingHole := range result {
+		switch votingHole {
+		case sebakcommon.VotingYES:
 			yes++
-		} else if vh == VotingNO {
+		case sebakcommon.VotingNO:
 			no++
```

Comment:
I changed to switch statement because sebakcommon.VotingEXP is added later.
If number of branches is 3 or more, then I think switch stmt is better.